fiks-meldingsformidler/src/main/java/no/difi/meldingsutveksling/ks/mapping/ForsendelseMapper.java

```diff
@@ -172,6 +273,36 @@ private Adresse mottakerFrom(InfoRecord infoRecord) {
         return mottaker.build();
     }
 
+    private List<Dokument> mapArkivmeldingDokumenter(ConversationResource cr, List<Object> docs) {
+        List<Dokument> dokumenter = Lists.newArrayList();
+
+        for (Object d : docs) {
+            if (d instanceof Dokumentbeskrivelse) {
+                Dokumentbeskrivelse db = (Dokumentbeskrivelse) d;
+                db.getDokumentobjekt().forEach(dbo -> {
+                    String f = dbo.getReferanseDokumentfil();
+                    try {
+                        byte[] bytes = persister.read(cr.getConversationId(), f);
+                        String[] split = dbo.getReferanseDokumentfil().split(".");
+                        String ext = Stream.of(split).reduce((p, e) -> e).orElse("pdf");
+                        String mimetype = MimeTypeExtensionMapper.getMimetype(ext);
+                        final DataHandler dataHandler = new DataHandler(new ByteArrayDataSource(new ByteArrayInputStream(bytes), mimetype));
+                        Dokument dokument = Dokument.builder()
+                                .withData(dataHandler)
+                                .withFilnavn(f)
+                                .withMimetype(mimetype)
+                                .build();
+                        dokumenter.add(dokument);
+                    } catch (IOException e) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1166)